runTest.sh

```diff
@@ -1,34 +1,92 @@
 #!/bin/bash
 
-# we store all the passwords in the local .env file
-echo -e '\n=> Exporting secrets'
-if [ -f .env ]
-then
-  # export "$(cat .env | xargs)"
-  source .env
-  # echo $DEVICE_CLOUD_USERNAME 
-else
-  echo -e '\n=> No env file found abborting operation'
-  echo "Missing .env file!" > logfile.log
+CLEAR='\033[0m'
+RED='\033[0;31m'
+
+function usage() {
+  if [ -n "$1" ]; then
+    echo -e "${RED}👉 $1${CLEAR}\n";
+  fi
+  manPage
+}
+
+function manPage() {
+  echo "Usage: $0 [-u upload-app] [-f fzf VALUE] [-r run]"
+  echo "  -u, --upload-app         Upload app to sauceLab, from your .env file"
+  echo "  -f, --fzf                Using FZF select your tests (SHIFT-TAB/TAB) to select and unlect"
+  echo "  -r, --run                Run the tests"
+  echo "  -v, --verbose            All the command prompts (not implemented!)"
+  echo ""
+  exit 1
+}
+
+function logClear(){
+  if [ $# -eq 0 ]; then
+    echo "[ERROR] - function input: logClear \$VALUE: file/path " 
+    exit 125
+  elif [[ -f "$1" ]]; then
+      echo -e 'cleaning tmp data file'
+    rm "$1"
+  fi
+  touch "$1"
+}
+
+function fzf() {
+
+  logClear .runTest
+  bash ./manage tests | rg -e '(@[^\s]+)' | uniq |  command fzf --border -m | tee .runTest
+  bat .runTest 
```

Comment:
@nodlesh I don't know how to parse the file to get only the uniques so that I may pass it to runTest()